evaltools/scoring/partisan.py

```diff
@@ -37,51 +37,55 @@ def _party_wins_by_district(part: Partition, election_cols: Iterable[str], party
     districts = [d for d in sorted(part.parts.keys()) if d != -1]
     return {d: int(d_wins) for d, d_wins in zip(districts, stability)}
 
+def _aggregate_seats(part: Partition, election_cols: Iterable[str], party: str) -> PlanWideScoreValue:
+    stability = _election_stability(part, tuple(election_cols), party)
+    return int(stability.sum())
+
 def _seats(part: Partition, election_cols: Iterable[str], party: str, mean: bool = False) -> ScoreValue:
-    result = {part[e].election.name: sum([part[e].won(party, d) for d in part.parts.keys() if d != -1]) for e in election_cols}
-    return np.mean(list(result.values())) if mean else result
+    result = {e: sum([part[e].won(party, d) for d in part.parts.keys() if d != -1]) for e in election_cols}
+    return int(np.mean(list(result.values()))) if mean else result
 
-def _signed_proportionality(part: Partition, election_cols: Iterable[str], party: str, mean: bool = False) -> ScoreValue:
-    result = {part[e].election.name: part[e].seats(party) - (part[e].percent(party) * len(part)) for e in election_cols}
-    return np.mean(list(result.values())) if mean else result
+def _responsive_proportionality(part: Partition, election_cols: Iterable[str], party: str) -> PlanWideScoreValue:
+    result = [(part[e].seats(party) / len(part)) - (part[e].percent(party)) for e in election_cols]
```

Comment:
In addition to this returning the mean signed proportionality instead of the by-election version (which I think is good!) it looks like it switches the units from seats to seat _shares_. Is this done to match the .NET version, or for some other reason? I guess it would be simple to translate back to seats, but in my mind it's a little bit easier to have this in terms of seats in the first place. What do you think?

Reply:
This does bring it inline with the .NET version.  Also from the conversation this morning it sounded like we wanted them to be between 0 and 1 (aka seat shares rather than raw seats), but I could be misremembering that.

Reply:
got it! yeah makes sense I'm good with that